packages/core/src/utilities/font-size.scss

```diff
@@ -10,9 +10,10 @@ Use the font size utility to make the text larger or smaller.
 
 Markup:
 <% ['display','title','h1','h2','h3', 'h4', 'lead','base','small'].forEach(size => { -%>
-<article class="ds-u-font-size--<%= size %> ds-u-leading--heading ds-u-margin-bottom--3 ds-u-truncate">
+<article class="ds-u-font-size--<%= size %> ds-u-leading--heading ds-u-margin-bottom--3 ds-u-truncate ds-u-border-bottom--1">
```

Comment:
When I first looked at this, the bottom border actually confused me about which set of variables belonged to which one. If you look at the top of the list you can tell, but it actually seemed like underlining a heading when my eye first hit it, which made me think the variables below it might belong to a heading. I think maybe an `<hr>` might be more appropriate because there would be some padding between it and the header before it.

Reply:
Ah, thanks for flagging. That was the opposite of what I was going for 😅  

I played around some more with this, how's this feel?

![image](https://user-images.githubusercontent.com/371943/36461815-a54e9160-168d-11e8-8075-ab1845bb1e30.png)

Reply:
That looks perfect! Not confusing at all 👍